playground/backend/internal/code_processing/code_processing.go

```diff
@@ -250,6 +308,55 @@ func cancelCheck(ctx context.Context, pipelineId uuid.UUID, cancelChannel chan b
 	}
 }
 
+// readLogFile reads logs from the log file and keeps it to the cache.
+// If context is done it means that the code processing was finished (successfully/with error/timeout). Write last logs to the cache.
+// If <-stopReadLogsChannel it means that the code processing was finished (canceled/timeout)
+// 	and it waits until the method stops the work to change status to the pb.Status_STATUS_FINISHED. Write last logs
```

Comment:
Why is there a tabulation necessary here?

Reply:
It is not necessary. Just wanted to separate it from other `If` cases.

Reply:
Agree with Aydar, it provides better readability